pkg/projects/Microsoft.NETCore.DotNetHostResolver/Microsoft.NETCore.DotNetHostResolver.props

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Version>$(HostResolverVersion)</Version>
```

Comment:
I find it odd that the `DotNetHostResolver` package is the only one that has this file. It isn't in the HostPolicy, nor the two Host packages.  Why is this one special?

I looked really quick, but I don't see where this file gets imported.
f
f